### PR TITLE
Add New Relic gem

### PR DESCRIPTION
**Pull request by JulianPasquale**
#### Description:

This PR adds the New Relic gem and it's config file.

---
#### Notes:
* The New Relic config file does not enable any of the optional configurations they provide. It only includes the minimum configs needed to start tracking the application metrics.
---

### CODE REVIEW

**Review by mrodriguez90 — APPROVED** [review]
LGTM

**Review by jeroig — APPROVED** [review]

**Review by jpascual1994 — APPROVED** [review]

File: .env.sample [hunk 1]
```diff
@@ -6,3 +6,5 @@ AWS_SECRET_ACCESS_KEY=
 S3_BUCKET_NAME=
 AWS_BUCKET_REGION=
 DOCKER_ENABLED=false
+NEW_RELIC_API_KEY=api_key
+NEW_RELIC_APP_NAME='Rails API base - development'
```
Comment (santib) [inline .env.sample:10]:
```suggestion
NEW_RELIC_APP_NAME='rails_api_base - development'
```

so `bootstrap.sh` replaces it

File: README.md [hunk 2]
```diff
@@ -99,6 +99,7 @@ To illustrate, `bin/rails console` will run the console in the docker container
 - [Simplecov](https://github.com/colszowka/simplecov) for code coverage
 - [Webmock](https://github.com/bblimke/webmock) for stubbing http requests
 - [YAAF](https://github.com/rootstrap/yaaf) for form objects
+- [New Relic](https://github.com/newrelic/newrelic-ruby-agent) for monitoring and debugging
```
Comment (santib) [inline README.md:102]: add alphabetically

File: README.md [hunk 3]
```diff
@@ -129,6 +130,11 @@ With `bundle exec rails code:analysis` you can run the code analysis tool, you c
 - [Rails Best Practices](https://github.com/flyerhzm/rails_best_practices#custom-configuration) Edit `config/rails_best_practices.yml`
 - [Brakeman](https://github.com/presidentbeef/brakeman) Run `brakeman -I` to generate `config/brakeman.ignore`
 
+## Monitoring
+
+In order to use [New Relic](https://newrelic.com) to monitor your application requests and metrics, you must setup `NEW_RELIC_API_KEY` and `NEW_RELIC_APP_NAME` environment variables.
```
Comment (enzofab91) [inline README.md:135]: Maybe?

```suggestion
In order to use [New Relic](https://newrelic.com) for monitoring your application requests and metrics, you must setup `NEW_RELIC_API_KEY` and `NEW_RELIC_APP_NAME` environment variables.
```

**Review by enzofab91 — APPROVED** [review]
🙌

File: README.md (hunk 3 above)
Comment (JulianPasquale) [reply README.md:135]: I think I like the infinitive version a bit more  😅

**Review by santib — APPROVED** [review]

**Pull request merged**